test/phoenix/integration/websocket_channels_test.exs

```diff
@@ -341,14 +341,14 @@ defmodule Phoenix.Integration.WebSocketChannelsTest do
                                       "path" => "/ws/connect_info/websocket",
                                       "query" => "vsn=#{@vsn}",
                                       "scheme" => "http",
-                                      "port" => 80}}}}
+                                      "port" => @port}}}}
```

Comment:
the commit message for 94a401951601d4848b4581031e743c98f095bcb5 has details on why this changed